docs/source/training_pipeline/01_getting_started.rst

```diff
@@ -0,0 +1,279 @@
+ Introduction 
+==============
+
+This section will introduce you to the training pipeline, its features, how it works, how you can configure the parameters and how you can train on your own dataset.
+PeekingDuck offers this feature for users who intend to perform their own model training.
+Training is possible without the need to write boilerplate codes. The training pipeline supports the image classification use case as of PeekingDuck version 2.0.
+
+
+Features
+--------
+
+.. raw:: html
+
+   <h6>TensorFlow and PyTorch Models</h6>
+Supports pre-trained models from both PyTorch and TensorFlow. The selection of the model can be customized in the configuration files.
+
+.. raw:: html
+
+   <h6>Train on custom dataset</h6>
+Configure the pipeline to train on your own dataset. You can refer to :ref:`using_custom_dataset` for more details.
+
+.. raw:: html
+
+   <h6>Analyze Runs</h6>
+The training pipeline uses Weights & Biases for tracking the performance, model versioning and visualizing the results. 
+The user has the option to setup an account on Weights & Biases to upload and view the training logs online, or on a local server to view the training logs.  Refer to the :ref:`setting-up-weights-and-biases` section below for more details.
+
+.. raw:: html
+
+   <h6>Configure Training Parameters</h6>
+You can configure the training parameters by changing the configuration yaml files or through the command line arguments. Refer to :ref:`configuring_training_parameters` for more details.
+
+.. raw:: html
+
+   <h6>Save the trained model</h6>
+The trained models will be saved in the output folder.
+
+
+
+How the training pipeline works
+===============================
+
+.. raw:: html
+
+   <h6>Overview Diagram</h6>
+
+#. Organize your data with the required format 
+#. Place them at the designated directory
+#. Personalise your training if necessary 
+#. Execute from the terminal
+
+**Component Diagram**
+
+.. image:: /assets/diagrams/C4Diagram-L4_SimplifiedOverview.png
+
+
+
+----------
+
+================
+ Getting Started
+================
+
+This guide explains how you can prepare your environment and test out the default pipeline using either TensorFlow or Pytorch.
+
+
+Install PeekingDuck Training Pipeline
+-------------------------------------
+
+.. tabs::
+
+   .. tab:: Linux
+
+      Plarform:
+         - Ubuntu 20.04 LTS / 22.04 LTS
+      Prerequisite:
+         - `Git <https://github.com/git-guides/install-git>`_
+         - `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_
+         - For CUDA GPU:
+            - `Latest Compatible Nvidia Display Driver for your system <https://www.nvidia.com/download/index.aspx>`_
+      
+      **CPU**
+
+      .. code-block:: bash
+         :linenos:
+
+         # Create conda environment & activate
+         conda create -n pkd-training python=3.8
+         conda activate pkd-training
+
+         # Clone PeekingDuck repository
+         git clone <TODO: PeekdingDuck repository>
```

Comment:
I think we can replace the `<TODO: ...>` with the actual PeekingDuck GitHub repo URL.

Reply:
replaced with peekingduck repo